docs/source/getting-started/index.rst

```diff
@@ -43,9 +57,12 @@ First off, open a terminal, clone the template project and move to the project f
     git clone https://github.com/digital-asset/create-daml-app.git
     cd create-daml-app
 
-In order to connect the DAML model to the UI code, we need to run a code generation step (more on this later)::
+Next we need to compile the DAML code to an archive ``.dar`` file::
 
     daml build
+
+In order to connect the UI code to this DAML, we need to run a code generation step (which we'll explain later)::
```

Comment:
I find the "I'll explain this later" stance problematic. Let's try to remove in a future iteration again.

Reply:
Removed